framework/src/media/MediaPlayerImpl.h

```diff
@@ -97,6 +97,7 @@ class MediaPlayerImpl : public std::enable_shared_from_this<MediaPlayerImpl>
 	std::shared_ptr<MediaPlayerObserverInterface> mPlayerObserver;
 	std::shared_ptr<stream::InputDataSource> mInputDataSource;
 	int mId;
+	uint8_t gOutCardId;
```

Comment:
The naming convention is different from other member variables. Please check this.

Reply:
Okay.
But the variable will be changed into 'audio_handler' by following the above Taejun's comment...